src/routes/hydra.ts

```diff
@@ -0,0 +1,355 @@
+import { NextFunction, Request, Response } from 'express'
```

Comment:
The diff is a bit difficult - are there code changes from the original one?

Reply:
Unchanged from hydra-integration branch with the exception of adapting to newer hydra/kratos npm API.

Reply:
Thank you for the clarification!